packages/jaeger-ui/src/actions/jaeger-api.test.js

```diff
@@ -174,20 +133,3 @@ describe('actions/jaeger-api', () => {
     expect(() => mock.verify()).not.toThrow();
   });
 });
-
-describe('allSettled', () => {
-  it('validate responses', async () => {
-    const res = await jaegerApiActions.allSettled([
-      Promise.resolve(1),
-      // eslint-disable-next-line prefer-promise-reject-errors
-      Promise.reject(2),
-      Promise.resolve(3),
-    ]);
-
-    expect(res).toEqual([
-      { status: 'fulfilled', value: 1 },
-      { status: 'rejected', reason: 2 },
-      { status: 'fulfilled', value: 3 },
-    ]);
-  });
```

Comment:
I removed this test as we have now deprecated the custom implementation of `allSettled` for the builtin `Promise.allSettled`.